### PR TITLE
[#155178552] Document how to respond to LogSearch queue alert

### PR DESCRIPTION
**Pull request by keymon**
[#155178552 Monitor Redis queue length for ELK stack](https://www.pivotaltracker.com/story/show/155178552)

What?
----

We added the monitoring of the Redis LogSearch queue size in [1]
and we want to document how to respond to this alert.

[1] https://github.com/alphagov/paas-cf/pull/1235

How to review?
-------

proof reading

Who?
----

Anyone but me

### CODE REVIEW

File: docs/support/responding_to_alerts.md [hunk 1]
```diff
@@ -186,6 +186,30 @@ fundimental connectivity issue to the reported endpoint.
 The `cf logs` output of the paas-metrics app may contain additional
 information. This is deployed to the 'monitoring' space of the 'admin' org.
 
+## Logsearch/ELK queue threshold limits
+
+We use [LogSearch (ELK
+Stack)](https://github.com/logsearch/logsearch-boshrelease) to process the CF
+logs.  LogSearch would accept messages from *ingestors* that store the raw
```
Comment (samcrang) [inline docs/support/responding_to_alerts.md]: It's a really small thing but I'd like to maintain consistency with upstream on how things are capitalised:

- Logsearch
- Elasticsearch
- Datadog

**Comment by keymon** [comment]
> I feel like this story doens't provide any strategies for dealing with the alerts—is it worth including something? Perhaps a method purging the existing logs so (at the very least) we get recent logs coming in?

I did not include any because I did not really knew any for sure :-D

I will ask @LeePorte

File: docs/support/responding_to_alerts.md (hunk 1 above)
Comment (keymon) [reply docs/support/responding_to_alerts.md]: :+1:

Comment (keymon) [reply docs/support/responding_to_alerts.md]: fixed

**Comment by LeePorte** [comment]
I agree with @samcrang. Some discussion should happen with @alext and @henrytk about actions to take regarding the queue and any deletion actions.

Reference can be made to the Elasticsearch CPU monitor notebook located at https://app.datadoghq.com/notebook/27407/Elasticsearch%20master%20CPU%20(copy) this provides an overview of the logging stack nodes and enables a quick view to see if we are suffering from CPU, Network or memory contention.

**Comment by keymon** [comment]
@samcrang I added some troubleshooting guidelines

**Pull request merged**